dev/com.ibm.ws.jaxws.tools/bnd.bnd

```diff
@@ -21,6 +21,8 @@ Export-Package: com.sun.tools.ws.ant;version="2.2.10", \
  com.ibm.jtc.jax.xml.ws.api;version=1.0.0, \
  com.ibm.jtc.jax.xml.ws.policy;version=1.0.0, \
  com.ibm.ws.jaxws.tools;version="2.2.10", \
+ org.jvnet.staxex;version="1.7.7", \
```

Comment:
This export is needed because this dependency is also required on Java 9+.  Since it was already used by the jaxws tools bundle I am simply exporting it from there rather than adding it again in another bundle.